meta/main.yml

```diff
@@ -0,0 +1,57 @@
+galaxy_info:
+  author: Sitewards & DnR-iData
+  description: Let's Encrypt roles
+  company: Sitewards
+
+  # If the issue tracker for your role is not on github, uncomment the
+  # next line and provide a value
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
+
+  min_ansible_version: 1.2
```

Comment:
This requires some RSA stuff only available in 2.4+

https://github.com/sitewards/ansible-role-lets-encrypt/blob/master/tasks/lets_encrypt.yml#L43
http://docs.ansible.com/ansible/latest/openssl_csr_module.html